tests/ert_tests/ert3/data/test_transformers.py

```diff
@@ -35,7 +34,7 @@ def record_factory(is_dir: bool):
             _files = [dir_path / "a.txt", dir_path / "b.txt"]
             with file_factory_context(tmpdir) as file_factory:
                 file_factory(_files)
-            return BlobRecord(data=path_to_bytes(dir_path))
+            return BlobRecord(data=_sync_path_to_bytes(dir_path))
```

Comment:
Is it really necessary to use the synchronous version?

Reply:
Probably not, but it looked slightly untrivial to rewrite the context managers in the test fixtures to be async.

Reply:
Reasonable, maybe a small comment with the reason and that it is sufficient?